roles/builder/defaults/main.yml

```diff
@@ -18,3 +18,5 @@ builder_compose_customizations:
     groups:
       - "users"
       - "wheel"
+builder_image_storage_threshold: 3 # percentage
```

Comment:
It's not clear to me what this variable's intended purpose is. Please add some information to the README. :)